<?xml version="1.0" encoding="UTF-8"?><testrun name="TestPostToMainPage" project="TestingFramework for 570.kz" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="my.test.TestPostToMainPage" time="45.445">
    <testcase name="test" classname="my.test.TestPostToMainPage" time="45.445">
      <error>org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.37.0', revision: 'a7c61cb', time: '2013-10-18 17:15:02'
System info: host: 'Tester1', ip: '192.168.1.42', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_21'
Driver info: driver.version: RemoteWebDriver&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:307)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByLinkText(RemoteWebDriver.java:356)&#13;
	at org.openqa.selenium.By$ByLinkText.findElement(By.java:248)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:299)&#13;
	at my.test.NewPostPage.GoTo(NewPostPage.java:14)&#13;
	at my.test.TestPostToMainPage.test(TestPostToMainPage.java:30)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
Caused by: java.net.ConnectException: Connection refused: connect&#13;
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)&#13;
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.PlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.SocksSocketImpl.connect(Unknown Source)&#13;
	at java.net.Socket.connect(Unknown Source)&#13;
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:117)&#13;
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:178)&#13;
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:144)&#13;
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:131)&#13;
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:610)&#13;
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:445)&#13;
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)&#13;
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)&#13;
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)&#13;
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:337)&#13;
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:298)&#13;
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:145)&#13;
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:366)&#13;
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:527)&#13;
	... 31 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
